est_frame: simulator_origin
control_frame: simulator_origin
N: 4
clearance: 0.5
max_vel: 30
rho_0: 3.0
rho_1 : 3.0
r : 0.4
m_soft : 1.5 
m_hard : 2.3263 
ros_rate : 10 
ts : 0.01
Kv : 0.25
lambda: 10
UAV_names: [uav1, uav2, uav3, uav4]
